chapters/synchronization.txt

```diff
@@ -3966,6 +3966,10 @@ semaphore).
     pname:commandBuffer was allocated from, as specified in the
     <<synchronization-pipeline-stages-supported, table of supported pipeline
     stages>>.
+  * [[VUID--srcQueueFamilyIndex-02803]]
```

Comment:
Where'd the VUID come from?

Reply:
This needs to be rebased. It was fixed in my PR.

Reply:
Just took care of this as I resolved the conflict.